### PR TITLE
MileStone_1 - 4 main sections

### PR DESCRIPTION
**Pull request by mahbubul14**
# TNW Responsive: 

This website is a mock-up of the original [website](https://perma.cc/M5ZV-Q2D6)

The main focus of this project is RWD (Responsive Web Design) and the tools we used includes:
- flexbox
- CSS grid
- media-query
- relative units(rem, em, %)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,587 @@
...
+							<a href="#"
+								><img
+									src="images/icons8-envelope-96.png"
+									alt="icons8-envelope-96"
+							/></a>
+						</li>
+					</ul>
+				</div>
+				<div class="footer-links">
+					<ul>
```
Comment (henatan99) [inline index.html:564]: use flex styling for this element

File: index.html [hunk 2]
```diff
@@ -0,0 +1,587 @@
...
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta
+			name="viewport"
+			content="width=device-width, initial-scale=1.0"
+		/>
+		<title>Perma | TNW</title>
+		<link rel="stylesheet" href="style.css" />
+	</head>
+	<body>
+		<nav class="d-flex">
```
Comment (henatan99) [inline index.html]: This nav tag should be changes to header tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,587 @@
...
+					<a href="#" class="d-flex">
+						<img
+							src="./images/logo-icon.png"
+							alt=""
+							class="logo"
+						/>
+					</a>
+				</div>
+				<div class="nav-top-R">
+					<div class="nav-top-row-1-container">
```
Comment (henatan99) [inline index.html]: This div tag should be changed to nav tag. Apply the same change for nav-top-row-2 class element too.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,980 @@
...
+  width: 100vw;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
+
+img {
+  width: 100%;
```
Comment (henatan99) [inline style.css:26]: Use absolute unit like px here. This one is distorting the logo image when the view width changes.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,587 @@
...
+								src="images/share-icon-white.png"
+								alt="share-icon"
+							/>5
+						</li>
+					</ul>
+				</header>
+			</article>
+		</div>
+		<section class="funding">
+			<header class="funding-header">
```
Comment (henatan99) [inline index.html]: don't use header tag here

File: index.html [hunk 6]
```diff
@@ -0,0 +1,587 @@
...
+				<h3>
+					<a href="#"
+						>Latest funding rounds on
+						Index.co</a
+					>
+				</h3>
+				<ul>
```
Comment (henatan99) [inline index.html]: wrap this ul element with nav tag

File: index.html [hunk 7]
```diff
@@ -0,0 +1,587 @@
...
+				</ul>
+			</header>
+			<div class="f-container">
+				<div class="f-first f-parent">
+					<img
+						class="funding-img"
+						src="./images/funding1.png"
+						alt=""
+					/>
+					<div class="p1">
```
Comment (henatan99) [inline index.html]: Use heading tag (hx) for all the titles of article elements in the latest funding section.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,587 @@
...
+						<li>
+							<a href="#"
+								><img
+									src="images/icons8-envelope-96.png"
+									alt="icons8-envelope-96"
+							/></a>
+						</li>
+					</ul>
+				</div>
+				<div class="footer-links">
```
Comment (henatan99) [inline index.html]: change this div tag to nav tag

File: index.html [hunk 9]
```diff
@@ -0,0 +1,587 @@
...
+							tripling revenues on
+							SiliconANGLE</a
+						>
+					</div>
+				</div>
+			</div>
+		</section>
+		<footer>
+			<div class="footer-row-1 d-flex">
+				<div class="footer-icon">
```
Comment (henatan99) [inline index.html]: change this div tag to nav tag

**Review by henatan99 — CHANGES_REQUESTED** [review]
# Changes Required :recycle:
Hi there, you have done a lot in this project :muscle: :+1: yet there are some changes to work on. Please go through all the review comments and make the necessary changes.

### Reminder: This is your first review and you will have only a maximum of two more reviews. If your project doesn't get approved in the third review you will be asked to repeat the project. So, make sure to make all the requested changes before you submit for re-review. 

## To Highlight 
- Nice to implement most of the business requirements
## To Improve [MANDATORY]
- You CSS file has a lot of unnecessary comments, please make your code clean. 
- You have repeated media queries for min-width 768px in your css file. Correct this one. 
- Footer links should be displayed in flex. Please see inline comment 
- Use header tag to wrap the header nav elements. See inline comments. 
- User proper nav tags for the header menu nav items. See inline comments 
- The TNW logo should appear on the first (big) article section as in the original website. See the attached screenshot image. 
TNW

- You haven's used the correct red overlay gradient background in the first article of the highlight sections for desktop view. Kindly change the range colors. 
- Date should appear over the first article of the highlights section for desktop view. 
- For proper semantic tags, see the inline comments and make changes accordingly. 

## [OPTIONAL] improvements
- You can improve on typography, but this change is not mandatory 
- Try as much as possible to make your page resembles the original one 

### Kindly make the requested changes and submit for re-review.

### Happy coding :computer: :computer_mouse: 

Should you have any query, find me via: 
[LinkedIn](https://www.linkedin.com/in/henok-mekonnen-2a251613/) | [Twitter](twitter.com/henatan99) | Henok Mekonnen on Slack

File: index.html [hunk 10]
```diff
@@ -0,0 +1,600 @@
...
+						</li>
+					</ul>
+				</header>
+			</article>
+			<article
+				class="third-highlight highlights-article d-flex"
+			>
+				<header>
+					<h2>
+						<a href="#"
```
Comment (MkrtichSargsyan) [inline index.html]: Don't use the header tag here.

**Review by MkrtichSargsyan — CHANGES_REQUESTED** [review]
## _Status:_ Changes Requested :recycle:

## Hello Guys

Good job on implementing the changes requested. However, there are some changes that you need to make in order to get your project approved


- [x] You still have a header tag inside the article

<hr>

## OPTIONAL

- [x] Add h1 tag to the page

  theoretically and logically wrong to have subheadlines ```<h2>``` without a main headline ```<h1>```,   but it won't have a negative effect.

- [x] you have a horizontal scrollbar on your page. You can fix that by removing ```width: 100vw;``` from the body


Please, make the REQUIRED changes and submit your request again! :grinning:
Good luck. 

> ps: . If you have any questions about my review or need help, you can message me on slack ( @mkrtichsargsyan )

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @mahbubul14 and @SamanAtashi :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**